### PR TITLE
CM-216: Update site footer

### PR DESCRIPTION
**Pull request by ayumi-oxd**
### Jira Ticket:
CM-216

### Jira Ticket URL:
https://bcparksdigital.atlassian.net/browse/CM-216

### Description:
- Changed link names and links based on the new design mock-up
- Removed the search function based on the new design mock-up
- Changed all `<Footer>{data.strapiWebsites.Footer}</Footer>` to `<Footer />` since the `Footer` component doesn't pass any data (`{data.strapiWebsites.Footer}` is not working as a prop)
- Delete `strapiWebsites` as this data has never been used in Gatsby

### CODE REVIEW

File: src/staging/__tests__/components/__snapshots__/footer.test.js.snap [hunk 1]
```diff
@@ -81,26 +52,46 @@ exports[`Footer renders correctly 1`] = `
...
+                      </a>
                     </div>
                   </li>
                   <li>
                     <div
                       className="mt-2"
                     >
                       <a
-                        href="/commercial-use"
+                        href="/commercial-use/filming-in-parks"
```
Comment (foley-lynn) [inline src/staging/__tests__/components/__snapshots__/footer.test.js.snap:73]: I think we're going with training slashes for internal urls, as this (and the others) currently re-direct. We'll have that chat next week to ensure that's correct.

File: src/staging/__tests__/components/__snapshots__/footer.test.js.snap [hunk 2]
```diff
@@ -228,10 +224,10 @@ exports[`Footer renders correctly 1`] = `
         </div>
       </div>
       <div
-        className="text-center py-3 mt-5 border-top border-white"
+        className="text-left text-sm-center py-3 mx-3 mx-md-0 mt-5 border-top border-white"
       >
         <div
-          className="footer-utility-link mx-1 mx-sm-2 mx-md-3 d-inline-block"
+          className="footer-utility-link my-2 mr-3 d-inline-block"
```
Comment (foley-lynn) [inline src/staging/__tests__/components/__snapshots__/footer.test.js.snap:230]: I think these links look a bit tight, I'd rather `mr-4`.

**Review by foley-lynn — APPROVED** [review]

**Review by nataliasabadysh — APPROVED** [review]

File: src/staging/__tests__/components/__snapshots__/footer.test.js.snap (hunk 1 above)
Comment (ayumi-oxd) [reply src/staging/__tests__/components/__snapshots__/footer.test.js.snap:73]: Sorry, I forgot to ask you about this. Do you know whether we should keep it as is or not?

Comment (foley-lynn) [reply src/staging/__tests__/components/__snapshots__/footer.test.js.snap:73]: Oh, leave it for now. I'll make a ticket to ensure we discuss it later.

**Pull request merged**